lib/std/array_list.zig

```diff
@@ -802,6 +825,55 @@ test "std.ArrayList/ArrayListUnmanaged.initCapacity" {
     }
 }
 
+test "std.ArrayList/ArrayListUnmanaged.clone" {
+    const a = testing.allocator;
+    {
+        var array = ArrayList(i32).init(a);
+        try array.append(-1);
+        try array.append(3);
+        try array.append(5);
+
+        const cloned = try array.clone();
+        defer cloned.deinit();
+
+        var i: usize = 0;
+        while (i < array.items.len) : (i += 1) {
+            try testing.expectEqual(array.items[i], cloned.items[i]);
+        }
+        try testing.expectEqual(array.items.len, cloned.items.len);
```

Comment:
Nitpick: this could be replaced by `try testing.expectEqualSlices(i32, array.items, cloned.items);`.

Reply:
Fixed 👍